playbooks/ovn.yaml

```diff
@@ -2,7 +2,6 @@
 - name: Deploy EDPM OVN
   hosts: all
   strategy: linear
-  become: true
```

Comment:
maybe this should not be removed... since we are getting permission denied in baremetal job.

Reply:
Either that or you need to add it to all the tasks where it's needed